extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -50,749 +141,685 @@ export class AjaxMonitor extends BaseTelemetryPlugin implements IDependenciesPlu
             enableCorsCorrelation: false,
             enableRequestHeaderTracking: false,
             enableResponseHeaderTracking: false,
-            enableAjaxErrorStatusText: false
+            enableAjaxErrorStatusText: false,
+            enableAjaxPerfTracking: false,
+            maxAjaxPerfLookupAttempts: 3,
+            ajaxPerfLookupDelay: 25
         }
         return config;
     }
 
     public static getEmptyConfig(): ICorrelationConfig {
-        return {
-            maxAjaxCallsPerView: undefined,
-            disableAjaxTracking: undefined,
-            disableFetchTracking: undefined,
-            disableCorrelationHeaders: undefined,
-            distributedTracingMode: undefined,
-            correlationHeaderExcludedDomains: undefined,
-            appId: undefined,
```

Comment:
This `appId` has no default value in `defaultConfig`, should it be added in `emptyConfig` as `undefined`?

Reply:
Nice catch!

Reply:
It will get set to undefined by the method lookup.
It needs to be here as the code that populates the config that is used also iterates over the keys